jobManager.py

```diff
@@ -17,6 +17,7 @@
 from config import Config
 from worker import Worker
 
+import tangoREST
```

Comment:
I accidentally introduced a circular dependency here. We should probably import tangod and call tangod#resetTango instead.